FactoryMod/config.yml

```diff
@@ -1522,6 +1521,19 @@ recipes:
       mycelium:
         material: MYCEL
         amount: 64
+  dirt_to_coarse_dirt:
```

Comment:
This needs forceInclude: true or it wont be added to the existing factories

Reply:
Done.